Source/Core/DolphinNoGUI/GLFW.h

```diff
@@ -0,0 +1,211 @@
+// Copyright 2016 Dolphin Emulator Project
```

Comment:
This file should probably have a cpp file that all the implementation details go into.